src/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -557,6 +562,7 @@ public void stopGame() {
     m_showHistoryAction = null;
     m_showMapOnlyAction = null;
     m_showCommentLogAction = null;
+    m_showUnitNationAction = null;
```

Comment:
let's do it. I've come across this code half a dozen times and have been tempted to as well.